tests/baselines/reference/blockScopedVariablesUseBeforeDef.types

```diff
@@ -70,12 +70,12 @@ function foo5() {
     let x = () => y;
 >x : () => () => any
 >() => y : () => () => any
->y : () => () => any
+>y : () => any

```

Comment:
This comes from:

```
function foo5() {
    let x = () => y;
    let y = () => x;
}
```

So, it's just where this infinite recursion ends; the tooltip says something else entirely too.